packages/go_router/lib/src/go_router_delegate.dart

```diff
@@ -174,6 +174,7 @@ class GoRouterDelegate extends RouterDelegate<Uri>
 
   /// Pop the top page off the GoRouter's page stack.
   void pop() {
+    log.info('popping $location');
```

Comment:
This should be in `go_router.dart`, where the other log statements are.

Reply:
For sure log statements was in `go_router_delegate.dart` when this PR was created 😉. Ok I will rebase it.